king_phisher/client/windows/plugin_manager.py

```diff
@@ -413,14 +427,23 @@ def signal_popup_menu_activate_reload(self, _):
 		for tree_iter in gui_utilities.gtk_treeview_selection_iterate(treeview):
 			model_row = self._model[tree_iter]
 			# only reloading installed plugins is currently supported
-			if model_row.parent is None:
-				continue
 			named_row = self._named_model(*model_row)
 			if named_row.type != _ROW_TYPE_PLUGIN:
```

Comment:
Can you make this block a bit more explicit. We have the type information already in the `named_row`, it seems like we should be using to check if it's a catalog and proceeding or if it's a repository find it's parent catalog and proceeding.